backend/pom.xml

```diff
@@ -132,7 +132,7 @@
     <dependency>
       <groupId>org.apache.kafka</groupId>
       <artifactId>connect-api</artifactId>
-      <version>3.2.3</version>
+      <version>3.4.0</version>
     </dependency>
```

Comment:
I'm curious if we shouldn't base this on the upstream debezium bom?

Reply:
Yes, but before we were diverging as we were behind the latest version or because of RHOC specifics.
Now we could make use of the debezium-bom. Could you file a Jira task for that?